helpers/getModalComponent.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  SignUpForm,
+  CheckEmail,
+  AccountActivated,
+  InvalidModal,
+} from 'components';
+
+const getModalComponent = (component: string | string[]) => {
+  if (component === 'signup') return SignUpForm;
```

Comment:
again, why this? why do u need this i do not understand, why can't u pass Componen itself